draft-ietf-cdni-uri-signing.xml

```diff
@@ -27,14 +27,14 @@
 
```

Comment:
I updated the other info in master already, but has the street address changed as well?